spring-cloud-dataflow-admin-local/src/main/java/org/springframework/cloud/dataflow/admin/spi/local/InProcessModuleDeployer.java

```diff
@@ -127,4 +130,13 @@ public ModuleStatus status(ModuleDeploymentId id) {
 		}
 		return statusMap;
 	}
-}
+
+	@Autowired
+	private Environment environment ;
```

Comment:
We should not be autowiring `Environment` into this class (especially not via field-injection).